Add initial template with projects. (#1)
WonderlandEngine/awesome-wonderland-engine: README.md

# Awesome Wonderland Engine [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)

> [<img src="https://yt3.googleusercontent.com/eZZOHhhAotLIcCXPNJ78lMrW-4wtrLJAg5ZqiEuqHkDMNKbBk3ZIHdVmfTdGEHlxmNcDrQVWFQ8=s900-c-k-c0x00ffffff-no-rj" align="right" width="100">](https://immersiveweb.dev/)

Reviewer: Is this the right URL?

Author: Commited the image instead.

> > A curated list of projects made with wonderland engine.

Reviewer: ```suggestion
> A curated list of projects made with Wonderland Engine.
```

[Wonderland Engine](https://wonderlandengine.com/) is a highly performant, accessible, and lightweight graphics engine tailored for the web and [WebXR](https://immersiveweb.dev/). It comes with a desktop editor application and an efficient WebAssembly-based runtime.

## Contents

-   [Awesome Wonderland Engine ](#awesome-wonderland-engine-)
    -   [Contents](#contents)
    -   [Official Resources](#official-resources)
    -   [Integrations](#integrations)
    -   [Featured Projects](#featured-projects)
    -   [Community Projects](#community-projects)
    -   [Contribute](#contribute)

## Official Resources

-   [Documentation](http://example.com)
-   [Written tutorials](https://wonderlandengine.com/tutorials/)

> -   [Youtube resources](https://www.youtube.com/c/WonderlandEngine)

Reviewer: ```suggestion
-   [Youtube Resources](https://www.youtube.com/c/WonderlandEngine)
```

I suggest you use title case everywhere like with "Cloud Examples", i.e. also fix "Screenshot Testing" to be consistent 🙏

-   [Examples](https://github.com/WonderlandEngine/wonderland-engine-examples)
-   [Cloud Examples](https://github.com/WonderlandEngine/wonderland-cloud-examples)
-   [Components](https://github.com/WonderlandEngine/components)
-   [Plugins](https://github.com/WonderlandEngine/plugins)
-   [Screenshot testing](https://github.com/WonderlandEngine/wonderland-screenshot-testing)

## Integrations

> -   [React UI](https://github.com/WonderlandEngine/wonderland-react-ui)

Reviewer: I would move this to a separate UI libraries section and add Canvas UI from Nik Lever and LazyWidgets from Playko

| mr NOT - enough IS enough              | Pipo            | My dearest puppet, did you miss me? I did. Why don't you STAND UP, SQUEEZE those buttons, and get ready to THROW? I am here for you, don't make me wait.                                                                                                                                                                                                                                 | https://heyvr.io/arcade/games/enough-is-enough           | Yes         | https://github.com/signorpipo/enough-is-enough |
| Mescolato                              | Pipo            | clear your mind                                                                                                                                                                                                                                                                                                                                                                          | https://signorpipo.itch.io/mescolato                     | Yes         | https://github.com/signorpipo/mescolato        |

> | De-Evil-ery                            | Elia & Marco    | That stupid Prince Ugo! He’s trying to wake up the Princess after you finally trapped her in the mystic blue sphere! Don’t panic! De-Evil-ery is the company that can support you in your fights! Destroy the mecha-birds sent by Prince Ugo to wake up the Princess, and gain evilness points to buy new weapons! Be Careful though! If the true love meter fills up… IT’S GAME OVER!!! | https://signorpipo.itch.io/de-evil-ery                   | Yes         | https://github.com/signorpipo/devilery         |

Reviewer: ```suggestion
| De-Evil-ery                            | Pipo & Marco    | That stupid Prince Ugo! He’s trying to wake up the Princess after you finally trapped her in the mystic blue sphere! Don’t panic! De-Evil-ery is the company that can support you in your fights! Destroy the mecha-birds sent by Prince Ugo to wake up the Princess, and gain evilness points to buy new weapons! Be Careful though! If the true love meter fills up… IT’S GAME OVER!!! | https://signorpipo.itch.io/de-evil-ery                   | Yes         | https://github.com/signorpipo/devilery         |
```

To be consistent. I wonder if it would be common on awesome-lists to link the authors?

| Sketchy                                | Pipo & Yinch    | Have you ever wanted to test if the new sofa will have enough place in the living room? Well, Sketchy is here to help you! Create, grab, scale, rotate, move, whatever! You can also share the room so that your best friend can approve the new arrangement Well, some use this to create whatever they want so, you can do that too!                                                   | https://signorpipo.itch.io/sketchy                       | Yes         | https://github.com/SignorPipo/sketchy          |
| PPlayground                            | Pipo            | Playground template project for the Wonderland Engine                                                                                                                                                                                                                                                                                                                                    | https://signorpipo.itch.io/pplayground-wonderland-engine | Yes         | https://github.com/signorpipo/wle-pplaygrounds |

> | Labyroots                              | I Polaretti     | The mother of all trees... After landing on earth, her spores began to spread, while the roots protected her heart. Many fought to defeat her, and to save our land, but they all turned into... It's time for you to venture into her maze, and put this madness to an end. The mother of all trees must succumb.                                                                       | https://heyvr.io/arcade/games/labyroots                  | Yes         | https://github.com/signorpipo/labyroots        |

Reviewer: ```suggestion
| Labyroots                              | Pipo & ??     | The mother of all trees... After landing on earth, her spores began to spread, while the roots protected her heart. Many fought to defeat her, and to save our land, but they all turned into... It's time for you to venture into her maze, and put this madness to an end. The mother of all trees must succumb.                                                                       | https://heyvr.io/arcade/games/labyroots                  | Yes         | https://github.com/signorpipo/labyroots        |
```

I guess their team name was I Polaretti, but maybe list the others of the team instead?
